packages/netsuite-adapter/src/transformer.ts

```diff
@@ -69,7 +69,8 @@ export const createInstanceElement = (customizationInfo: CustomizationInfo, type
         [OBJECT_NAME]: type.elemID.getFullName(),
       }),
     }
-    const desiredName = naclCase(transformedValues[serviceIdFieldName])
+    const desiredName = naclCase(transformedValues[serviceIdFieldName]
+      .slice(isCustomType(type) ? 0 : FILE_CABINET_PATH_SEPARATOR.length))
```

Comment:
would it be better to check the real thing you want to check (if the path starts with `/`) rather than check `isCustomType` here?.
maybe this can even be just
```suggestion
const desiredName = naclCase(transformedValues[serviceIdFieldName]).replace(/^\//, '')
```
wdyt?

Reply:
I guess you meant that the `replace` would be on `transformedValues[serviceIdFieldName]`.
It would work as well and if you vote for it I have no objection..
Will modify it only to use FILE_CABINET_PATH_SEPARATOR

Reply:
yep, my mistake.
it doesn't have to be the "replace" solution, the point of the comment is to check the condition that matters, in this case, the replace or slice should happen if the first character is the `FILE_CABINET_PATH_SEPARATOR` so that should be the condition instead of `isCustomType` IMO